File: ifthenelseif.pas
{ ifthenelseif.pas }
program IfThenElseIf;
var
  x: integer;
begin
  x := 5;
  if (x = 5) then
    writeln('x is 5')
  else if (x < 10) then
    writeln('x is less than 10');
end.

File: ifthenelseif.f90
! ifthenelseif.f90
program ifthenelseif
    implicit none
    integer :: x
    x = 5

    if (x == 5) then
        print *, 'x is 5'
    else if (x < 10) then
        print *, 'x is less than 10'
    end if
end program ifthenelseif
